src/test/suite/explorer/schemaTreeItem.test.ts

```diff
@@ -32,7 +32,7 @@ suite('SchemaTreeItem Test Suite', function () {
     const testSchemaTreeItem = new SchemaTreeItem(
       'cheesePizza',
       TEST_DB_NAME,
-      {},
+      {} as any,
```

Comment:
Can we import `DataService` type here and do `{} as DataService` to avoid `any`?

Reply:
Does it error for you? I tried `import type { DataService } from 'mongodb-data-service';` and `{} as DataService,` and this works well for me.